sandbox/demoapp/CreateChatView.qml

```diff
@@ -73,26 +74,43 @@ Page {
         }
 
         StatusButton {
+            implicitWidth: 106
             implicitHeight: 44
             Layout.alignment: Qt.AlignTop
             enabled: (tagSelector.namesModel.count > 0)
             text: "Confirm"
         }
+
+        Item {
+            implicitHeight: 32
+            implicitWidth: 32
+            Layout.alignment: Qt.AlignTop
+
+            StatusActivityCenterButton {
+                id: notificationButton
+                anchors.right: parent.right
+                unreadNotificationsCount: 3
+            }
+        }
     }
 
     contentItem: Item {
         anchors.fill: parent
 
         StatusBaseText {
-            visible: (contactsModel.count === 0)
-            anchors.centerIn: parent
+            anchors.left: parent.left
+            anchors.leftMargin: 252*0.66
```

Comment:
Any  reason we're calculating this (despite those numbers looking kinda odd), instead of hardcoding the result?
The values don't seem variable so just curious why we're doing it this way

Reply:
right, it was there from before in the width property, so I moved it to the margis but it's not needed as it seems! removed :)